extensions/wikia/PortableInfobox/js/PortableInfobox.js

```diff
@@ -29,18 +29,14 @@
 
 	var CollapsibleGroup = {
 		init: function() {
-			var $collapsibleGroups = $('.pi-collapse');
-
-			$collapsibleGroups.each( function( index, group ) {
-				var $group = $collapsibleGroups.eq(index),
-					$header = $group.find('.pi-header');
-
-				$header.click( function() {
-					$group.toggleClass('pi-collapse-closed');
-				});
-			});
+			$('.pi-header').click( function( e ) {
```

Comment:
I think my only issue with this is that you are binding events for all .pi-headers, even if they don't belong to a 
`.pi-collapse` group. Might work better to do something like `$('.pi-collapse .pi-header').click(...`